source/db/ar-projects.json

```diff
@@ -3461,7 +3461,6 @@
       "S/MIME",
       "SSL/TLS",
       "Tor",
```

Comment:
@vyp Sorry. I missed that... Fixed with `sed -i -e ':a;N;$!ba;s/Tor\",\n\ \ \ \ ],/Tor\"\n\ \ \ \ ],/' *-projects.json`.